cmd/restic/cmd_recover.go

```diff
@@ -24,15 +25,29 @@ Exit status is 0 if the command was successful, and non-zero if there was any er
 `,
 	DisableAutoGenTag: true,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		return runRecover(globalOptions)
+		return runRecover(globalOptions, recoverOptions)
 	},
 }
 
+type RecoverOptions struct {
+	All     bool
+	Combine bool
+	DryRun  bool
+	Tags    []string
+}
+
+var recoverOptions RecoverOptions
+
 func init() {
 	cmdRoot.AddCommand(cmdRecover)
+	recoverFlags := cmdRecover.Flags()
+	recoverFlags.BoolVar(&recoverOptions.All, "all", false, "add all directories, not only unreferenced ones")
+	recoverFlags.BoolVar(&recoverOptions.Combine, "combine", true, "if true, combine all directories into one snapshot, else generate multple snapshots")
+	recoverFlags.BoolVarP(&recoverOptions.DryRun, "dry-run", "n", false, "only show what would be done")
```

Comment:
`--dry-run` only seems to be useful in combination with `--combine=false`?

Reply:
Another use case would be to use it for debugging/troubleshooting which would also work on read-only repositories.